README.md

```diff
@@ -47,22 +47,46 @@ From your shell or command line:
 ### [OPTIONAL] Step 2:  Register the sample on the app registration portal
 
 You can run the sample as is with its current settings, or you can optionally register it as a new application under your own developer account.
-Create a new app at [apps.dev.microsoft.com](https://apps.dev.microsoft.com), or follow these [detailed steps](https://azure.microsoft.com/en-us/documentation/articles/active-directory-v2-app-registration/).  Make sure to:
 
-- Copy down the **Application Id** assigned to your app, you'll need it in the next optional step.
-- Add the **Native Application** platform for your app.
+#### Choose the Azure AD tenant where you want to create your applications
```

Comment:
Will MSA people know this is an AAD tenant?

Reply:
That's a good question. what would you recommend?

Reply:
"Choose the directory for your apps"

Reply:
@danieldobalian : I get your point, but if I don't know anything about identity, I might think that directory means folder? or not? (I would have thought so when I joint)